src/codeu/chat/database/Database.java

```diff
@@ -0,0 +1,55 @@
+package codeu.chat.database;
+
+import java.sql.*;
+
+import codeu.chat.util.Logger;
+
+public final class Database {
+
+  private final static Logger.Log LOG = Logger.newLog(Database.class);
+
+  private String m_dbPath;
```

Comment:
It is a good practice to stick with a single variable naming convention across all files. I think most of the existing code doesn't use the m_ prefix for member variables. So might be easier to follow that.